drivers/backup/parse_cloud_config.go

```diff
@@ -0,0 +1,133 @@
+package backup
```

Comment:
Rename this file to `cloud_cred_config.go`

Reply:
done